README.md

```diff
@@ -86,6 +87,61 @@ lk project list
 lk project set-default <project_name>
 ```
 
+## Bootstrapping an application
+
+The LiveKit CLI can help you bootstrap applications from a number of convenient template repositories, using your project credentials to set up required environment variables and other configuration automatically. To create an application from a template, run the following:
+
+```shell
+lk app create --template <template_name> my-app
+```
+
+Then follow the CLI prompts to finish your setup.
+
+The `--template` flag may be omitted to see a list of all available templates, or can be chosen from a selection of our first-party templates:
+
+<table>
+  <thead><tr><th>Template Name</th><th>Language/Framework</th><th>Description</th></tr></thead>
+  <tbody>
+    <tr>
```

Comment:
is it possible to make this markdown? would make adding entries much easier.

Reply:
actually, do you think we need to list them? or is there a mode to print this table from livekit-cli ? `lk app list-templates`

Reply:
This sound much better!